packages/node_modules/@ciscospark/internal-plugin-encryption/src/kms.js

```diff
@@ -401,7 +406,7 @@ const KMS = SparkPlugin.extend({
       .catch((reason) => {
         if (process.env.NODE_ENV === 'test' && (reason.status === 403 || reason.statusCode === 403) && reason.message.match(/Failed to resolve authorization token in KmsMessage request for user/)) {
           this.logger.warn('kms: rerequested key due to test-mode kms auth failure');
-          return this.request(payload);
+          return this.request(payload, undefined, onBehalfOf);
```

Comment:
This is a bit awkward. It's probably best to create the second argument as an options object with `timeout` and `onBehalfOf`.

Reply:
Good call @mattnorris - I've switched both the KMS and Encryption APIs over to use an option object.

Reply:
Any other change you'd like to see @mattnorris?

Reply:
Looks good, @gannons

Reply:
Perfect @mattnorris - how does a merge work on the sdk? Is it scheduled or do people commit adhoc?